packages/grafana-ui/src/components/Collapse/CollapsableSection.story.tsx

```diff
@@ -15,7 +15,7 @@ export default {
 export const simple = () => {
   return (
     <CollapsableSection label="Collapsable section" isOpen>
-      <div>Here's some content</div>
+      <div>{"Here's some content"}</div>
```

Comment:
Ref https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md

Reply:
I think this will make the code more cluttered if we'll need to escape all text inside JSX like this.

Reply:
Well, it's clutter vs convention. In this case we could just use the &apos; entity instead. We can disable this rule tho if people find it problematic. We can also adjust the rule settings to ignore ' and " https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md#forbid